packages/poc-cra-template-external-non-redux/template/config-overrides.js

```diff
@@ -0,0 +1,53 @@
+const { override, addBabelPreset } = require('customize-cra')
+
+/**
+ * Add linaria loader after babel loader
+ */
+function transformLoader(loader) {
+  const options = loader.options || {}
+  const presets = options.presets || []
+  options.presets = presets
+
+  return {
+    test: loader.test,
+    include: loader.include,
+    rules: [
+      {
+        loader: loader.loader,
+        options,
+      },
+      {
+        loader: 'linaria/loader',
+        options: {
+          cacheDirectory: 'src/.linaria_cache',
+          sourceMap: process.env.NODE_ENV !== 'production',
+          babelOptions: {
+            presets,
+          },
+        },
+      },
+    ],
+  }
+}
+
+const addLinariaLoader = (config) => {
+  /**
+   * cra scripts rules atm (version 3)
+   * https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.js
+   */
+  const ruleWithOneOf = config.module.rules.find((rule) => rule.oneOf)
```

Comment:
Will need to move override to separated package incase `react-scripts` config change